services/yaml.py

```diff
@@ -39,11 +39,21 @@ def fetch_commit_yaml(commit: Commit, owner: Owner | None) -> Dict | None:
         # have various exceptions, which we do not care about to get the final
         # yaml used for a commit, as any error here, the codecov.yaml would not
         # be used, so we return None here
+
+        if isinstance(owner, Owner):
+            owner_arg = owner.ownerid
+        elif isinstance(owner, User):
+            owner_arg = owner.pk
+        elif owner is not None:
+            owner_arg = str(type(owner))
+        else:
+            owner_arg = "None"
+
         log.warning(
             f"Was not able to fetch yaml file for commit. Ignoring error and returning None. Exception: {e}",
             extra={
                 "commit_id": commit.commitid,
-                "owner": owner.ownerid if owner else None,
+                "owner_arg": owner_arg,
```

Comment:
while we're logging things, do you want to log the object type? so when you have the id you know which table to `.get()` from?